### PR TITLE
uwsgi: move to Python 3

### PR DESCRIPTION
**Pull request by fxcoudert**
Python 2 is going away soon… https://github.com/Homebrew/homebrew-core/issues/47050

### CODE REVIEW

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

File: Formula/uwsgi.rb [hunk 1]
```diff
@@ -26,16 +26,9 @@ class Uwsgi < Formula
...
 
-  # "no such file or directory: '... libpython2.7.a'"
-  # Reported 23 Jun 2016: https://github.com/unbit/uwsgi/issues/1299
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/726bff4/uwsgi/libpython-tbd-xcode-sdk.diff"
-    sha256 "d71c879774b32424b5a9051ff47d3ae6e005412e9214675d806857ec906f9336"
-  end
-
   def install
     # Fix file not found errors for /usr/lib/system/libsystem_symptoms.dylib and
```
Comment (bayandin) [inline Formula/uwsgi.rb]: Since homebrew doesn't support Sierra and El Capitan, do we still need this?

Comment (fxcoudert) [reply Formula/uwsgi.rb]: For older versions: we don't actively remove things that make them work. We don't build bottles, and we don't offer support, but we will keep them working if it has no cost for us, basically.

Comment (bayandin) [reply Formula/uwsgi.rb]: Got it, thanks for the explanation!

**Pull request closed without merging**